crates/egui/src/containers/popup.rs

```diff
@@ -327,13 +342,19 @@ pub fn popup_above_or_below_widget<R>(
                         .inner
                     })
                     .inner
-            })
-            .inner;
+            });
+
+        let is_close = match close_behavior {
```

Comment:
```suggestion
        let should_close = match close_behavior {
```

("is close" sounds like it is about proximity)